pex/cli/commands/lock.py

```diff
@@ -657,6 +674,17 @@ def _add_export_arguments(
             type=ExportSortBy.for_value,
             help="How to sort the requirements in the export (if supported).",
         )
+        export_parser.add_argument(
+            "--include-dependency-info",
+            "--no-include-dependency-info",
```

Comment:
This is a bit of a hack around https://github.com/astral-sh/uv/issues/13383 to enable interop tests now with uv. I seem no harm in supporting this option forevermore though.